authentication/auth-api.js

```diff
@@ -0,0 +1,57 @@
+// Generic authentication script for authing against an API
+if (typeof println == 'undefined') this.println = print;
```

Comment:
This workaround can be removed, `println` is not being used (ditto for the other scripts, the one that's using it can be changed to just use `print`). If we happen to need the workaround again we should implement it in core instead of requiring all scripts to add it.